docs/openapi/components/schemas/common/SeaCargoManifest.yml

```diff
@@ -13,7 +13,8 @@ properties:
           enum:
             - SeaCargoManifest
       - type: string
-        const: SeaCargoManifest 
+        const: 
+          - SeaCargoManifest 
```

Comment:
is this right?

Reply:
I don't think it matters. This is what BL and CI are structured.